src/main.py

```diff
@@ -126,6 +126,7 @@ def put(self):
         resources = []
         for resource in data:
             resources.append(DpResource(
+                # TODO type=resource.get('type'),
```

Comment:
Okej, to ja jutro tutaj dodam sprawdzanie typu i zwracanie odpowiedniego z REPO_TYPES.